CONTRIBUTING.md

```diff
@@ -212,6 +212,10 @@ IDEA](https://www.jetbrains.com/idea/) and set up a new project. If you already
 have IntelliJ installed, please ensure you using version 2021.3.1 or later.
 We've had reports of build failures with earlier versions.
 
+For some tests to succeed, you will have to [set the environment
+variable](https://www.jetbrains.com/help/objc/add-environment-variables-and-program-arguments.html#add-environment-variables)
+`APALACHE_HOME` to the root of the Apalache source tree.
```

Comment:
This is not accurate, afaik. No tests should require this. This is just about debugging practices used in development, not about testing.

Reply:
Some unit tests have always required setting `-DTLA-Library` when run from IDEA, see [TestSanyImporterStandardModules.scala](https://github.com/informalsystems/apalache/pull/1553/files/998f95cd75bdcc33bcd0093224b5114cfb2b26fc#diff-3a0cbae0ae23361e326f995194bf7a9d207190b62f666d3eeb5ef75943bc4ccd)

Reply:
Yes, I see that now, thank you. I think this concern is subsumed by my view that those tests should be moved into integration tests.